### PR TITLE
feat: partnered orgs added to context

### PR DESCRIPTION
**Pull request by benstoltz**
1. Description: adds partneredOrgs and partneredOrgIds to context.

1. Instructions for testing:

1. Closes Issues: #<number> (if appropriate)

1. [x] Updated meaningful TSDoc to methods including Parameters and Returns, see [Documentation Guide](https://esri.github.io/hub-components/storybook/?path=/story/guides-documentation--page)

1. [x] used semantic commit messages
  
1. [x] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

1. [x] updated `peerDependencies` as needed. **CRITICAL** our automated release system can **not** be counted on to update `peerDependencies` so we _must_ do it manually in our PRs when needed. See the [updating peerDependencies](/RELEASE.md#Updating-peerDependencies) section of the release instructions for more details.

### CODE REVIEW

**Comment by sonofflynn89** [comment]
@benstoltz please don't merge this until I figure out what's wrong with `14.13.0`, sorry for the inconvenience 😔

File: packages/common/src/types.ts [hunk 1]
```diff
@@ -123,6 +123,21 @@ export interface IHubUserRequestOptions extends IHubRequestOptions {
   authentication: UserSession;
 }
 
+export interface IHubTrustedOrgsResponse {
```
Comment (dbouwman) [inline packages/common/src/types.ts:126]: Doc comments on the interface and all the props :)

File: packages/common/src/ArcGISContextManager.ts [hunk 2]
```diff
@@ -390,6 +434,32 @@ function getServiceStatus(portalUrl: string): Promise<HubServiceStatus> {
...
 }
 
+/**
+ * Get trusted orgs w/ a failSafe to return an empty array
+ */
+async function getTrustedOrgs(
+  _portalUrl: string,
+  _authentication: UserSession
+): Promise<IHubTrustedOrgsResponse[]> {
+  const failSafeTrustedOrgs = failSafe(request, { trustedOrgs: [] });
```
Comment (dbouwman) [inline packages/common/src/ArcGISContextManager.ts:444]: ❤️ failsafe ftw

**Review by dbouwman — APPROVED** [review]
This looks good to me - did you test this by yalc/copying into hub-components? Or add something into the context-manager.e2e.ts? 

Always good to do quick verification against the live api before merging, and then finding you need to make a breaking change :)

**Pull request merged**